src/content/plugins/ignore-plugin.md

```diff
@@ -18,8 +19,28 @@ The following examples demonstrate a few ways this plugin can be used.
 
 ## Ignore Moment Locales
 
-As of [moment](https://momentjs.com/) 2.18, all locales are bundled together with the core library (see [this GitHub issue](https://github.com/moment/moment/issues/2373)). You can use the `IgnorePlugin` to stop any locale being bundled with moment:
+As of [moment](https://momentjs.com/) 2.18, all locales are bundled together with the core library (see [this GitHub issue](https://github.com/moment/moment/issues/2373)). 
+
+Contrary to what you may guess, The `requestRegExp` parameter is not tested against the resolved file names or absolute module
```

Comment:
these line breaks within a paragraph should be removed imo

Reply:
Dont jump into assumption on what the reader might guess or not, this needs rephrasing

Reply:
@EugeneHlushko: removed line breaks and did some rewording.